### PR TITLE
#170947430 Setup docker

### PR DESCRIPTION
**Pull request by RusimbiPatrick**
#### What does this PR do?
Have Docker configured
#### Description of Task to be completed?

- Setting up Dockerfile
- Setting up docker-compose
- Creating a script for running docker

#### How should this be manually tested?

- Install docker on a local machine
- run `npm run docker`

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[Pivotal tracker](https://www.pivotaltracker.com/story/show/170947430)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        fi
+
+# Bundle app source inside the docker image
+COPY . .
+
+# Expose our app port inside the app and 
+EXPOSE 3000
+
+# Define commands that will run the app
+CMD ["npm", "start"]
```
Comment (AbdoulNuru) [inline Dockerfile]: @RusimbiP Add a new Line at the end of the file

**Review by AbdoulNuru — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (RusimbiP) [reply Dockerfile]: > @RusimbiP Add a new Line at the end of the file

Thanks

Comment (RusimbiPatrick) [reply Dockerfile]: Thanks

**Review by blaise82 — APPROVED** [review]

**Review by hakoemmy — APPROVED** [review]

**Review by izzett222 — APPROVED** [review]

File: .dockerignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+node_modules
+npm-debug.log
```
Comment (kcharles52) [inline .dockerignore:2]:
```suggestion
npm-debug.log

```

**Review by kcharles52 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**